Dockerfile

```diff
@@ -1,12 +1,11 @@
-# Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
-ARG APP_INSIGHTS_AGENT_VERSION=2.4.1
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.0
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
 LABEL maintainer="https://github.com/hmcts/ccd-data-store-api"
 
 COPY build/libs/core-case-data.jar /opt/app/
-COPY lib/applicationinsights-agent-2.4.1.jar lib/AI-Agent.xml /opt/app/
```

Comment:
TODO: Check that App Insights still works on latest version, and relying on the HMCTS-provided implementation rather than the jar that was in the repo.